openstudiocore/src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -174,31 +185,37 @@ namespace openstudio{
       if (m_connectionOpen)
       {
         try {
-          execAndThrowOnError("CREATE INDEX rmdTI ON ReportMeterData (TimeIndex ASC);");

```

Comment:
@mbadams5 seems like we could switch on E+ version here, these queries will all fail on e+ 8.2 or earlier